### PR TITLE
Start building out Modal display

### PR DESCRIPTION
**Pull request by ryancmoniz**

### CODE REVIEW

File: client/src/components/ModalBody.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+`
+
+const AlbumThumb = styled.div`
+  background-color: grey;
+  width: 100%;
+  margin:0;
+  margin-bottom: 8px;
+  min-width: 120px;
+  max-width: 140px;
+  height: 92px;
```
Comment (ingridclaire) [inline client/src/components/ModalBody.jsx:35]: So I'm assuming this is the syntax for setting up a styled component? 
const componentName = styled.[element]`css properties`

File: client/src/components/ModalBody.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        <AlbumThumb></AlbumThumb>
+        <AlbumThumb></AlbumThumb>
+        <AlbumThumb></AlbumThumb>
+        <AlbumThumb></AlbumThumb>
+        <AlbumThumb></AlbumThumb>
+      </Albums>
+      <Gallery>
+        <GalleryHeader>Room & Suite</GalleryHeader>
+      </Gallery>
+    </Container>
```
Comment (ingridclaire) [inline client/src/components/ModalBody.jsx:66]: And then everything goes inside the Container div? Where are you exporting this to?

File: client/src/components/ModalHeader.jsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+`
+
+function ModalHeader({hideModal}) {
+  return (
+    <Header>
+      <Title>Photos of H20 Suites</Title>
+      <Hotel>TripAdvisor</Hotel>
+      <Price>$349</Price>
+      <DealButton>View Deal</DealButton>
+      <CloseButton onClick={hideModal}>X</CloseButton>
```
Comment (ingridclaire) [inline client/src/components/ModalHeader.jsx:72]: Is this just how modals work in general? you can use {hideModal} onClick? or is that a function you set up?

File: client/src/components/ModalBody.jsx (hunk 1 above)
Comment (ryancmoniz) [reply client/src/components/ModalBody.jsx:35]: Yes, exactly!

**Pull request merged**